content/docs/reactivesearch/v3/advanced/stateprovider.md

```diff
@@ -77,7 +77,7 @@ For example:
 ```
 
 -   **includeKeys** `string[]` [optional]
-    defaults set to `['value', 'hits', 'aggregations', 'error']` which means that by default your search state for a particular component will only contains these keys. Although the default search state fulfills most of your common use cases but you can also control it by defining your custom keys with the help of `includeKeys` prop.<br/><br/>
+    Allowed values are `['value', 'hits', 'aggregations', 'error', 'promotedData', 'resultStats']`. Defaults set to`['value', 'hits', 'aggregations', 'error']`which means that by default your search state for a particular component will only contains these keys. Although the default search state fulfills most of your common use cases but you can also control it by defining your custom keys with the help of`includeKeys` prop.<br/><br/>
```

Comment:
@ShahAnuj2610 Remove new keys from here and Add the keys here https://i.imgur.com/LFIZgjT.png